frontend/test/specs/inviteInstitutionControllerSpec.js

```diff
@@ -23,19 +23,30 @@
         invites:[]
     };
 
+    var request = {
+        admin_name: 'example',
+        institution_key: splab.key,
```

Comment:
Don't use specific names to create object mocks, always use generic names to provide a general environment of tests.